kale/embed/video_i3d.py

```diff
@@ -9,6 +14,8 @@
 import torch.nn.functional as F
 from torchvision.models.utils import load_state_dict_from_url
 
+__all__ = ["i3d_joint", "InceptionI3d", "InceptionModule"]
```

Comment:
What is the purpose of defining `__all__`? 

Wildcard import `from ... import *` is [considered as a bad practice](https://docs.python.org/3/tutorial/modules.html#importing-from-a-package) and we should not use it. Therefore, defining `__all__` is not useful, right?

I know some other code did it but those are now considered as bad practice by python, as least according to what I found out.